amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/action/RecordsManagementActionServiceImpl.java

```diff
@@ -78,6 +80,20 @@
     private ClassPolicyDelegate<BeforeRMActionExecution> beforeRMActionExecutionDelegate;
     private ClassPolicyDelegate<OnRMActionExecution> onRMActionExecutionDelegate;
 
+    private DispositionService dispositionService;
+
+    public void setDispositionService(DispositionService dispositionService) {
+        this.dispositionService = dispositionService;
+    }
+
+
+    /** list of disposition actions to automatically execute */
+    private List<String> retentionActions;
+
+    public void setRetentionActions(List<String> retentionActions) {
```

Comment:
[Formatting] Braces should be on their own line.

There's an ide-config directory containing Eclipse and IntelliJ settings to help ensure our formatting is consistent.  I think over time we've noticed issues with these config files, and we also know there are plenty of existing files not formatted correctly.  However I'd be happy to review/approve fixes to these config files.